site/content/en/latest/design/backend-traffic-policy.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "BackendTrafficPolicy "
+---
+
+## Overview
+
+This design document introduces the `BackendTrafficPolicy` API allowing users to configure
+the behavior for how the Envoy Proxy server communicates with upstream backend services/endpoints.
+
+## Goals
+
+- Add an API definition to hold settings for configuring behavior of the connection between the backend services
+and Envoy Proxy listener.
+
+## Non Goals
+
+- Define the API configuration fields in this API.
+
+## Implementation
+
+`BackendTrafficPolicy` is an implied hierarchy type API that can be used to extend [Gateway API][].
+It can target either a `Gateway`, or an xRoute (`HTTPRoute`/`GRPCRoute`/etc.). When targeting a `Gateway`,
+it will apply the configured settings within ght `BackendTrafficPolicy` to all children xRoute resources of that `Gateway`.
+If a `BackendTrafficPolicy` targets an xRoute and a different `BackendTrafficPolicy` targets the `Gateway` that route belongs to,
+then the configuration from the policy that is targeting the xRoute resource will win in a conflict.
+
+### Example
+
+Here is an example highlighting how a user can configure this API.
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: GatewayClass
+metadata:
+  name: eg
+spec:
+  controllerName: gateway.envoyproxy.io/gatewayclass-controller
+---
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: eg
+  namespace: default
+spec:
+  gatewayClassName: eg
+  listeners:
+    - name: http
+      protocol: HTTP
+      port: 80
+---
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: ipv4-route
+  namespace: default
+spec:
+  parentRefs:
+    - name: eg
+  hostnames:
+    - "www.foo.example.com"
+  rules:
+    - backendRefs:
+        - group: ""
+          kind: Service
+          name: ipv4-service
+          port: 3000
+          weight: 1
+      matches:
+        - path:
+            type: PathPrefix
+            value: /
+---
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: ipv6-route
+  namespace: default
+spec:
+  parentRefs:
+    - name: eg
+  hostnames:
+    - "www.bar.example.com"
+  rules:
+    - backendRefs:
+        - group: ""
+          kind: Service
+          name: ipv6-service
+          port: 3000
+          weight: 1
+      matches:
+        - path:
+            type: PathPrefix
+            value: /
+---
+apiVersion: gateway.envoyproxy.io/v1alpha1
+kind: BackendTrafficPolicy
+metadata:
+  name: default-ipv-policy
+  namespace: default
+spec:
+  protocols:
```

Comment:
can we use a different feature as an example, maybe load balancing
ipv6 routing is tied to L3 which EG cannot really control, it can only make sure that if ipv6 endpoints exist, it can successfully fetch it and program it